### ISSUE TITLE
The max bandwidth label is confusing

### ISSUE THREAD

**Issue by bassosimone** [m-lab/ndt7-client-javascript#6]
As reported by @stephen-soltesz, naming something `max foo` leads one to expect that this quantity cannot decrease. Yet, that's the BBR's view of the max bandwidth, which is not the same of saying this is the max over the test lifetime. A better label would help.

Cc: @elioqoshi @georgiamoon

**Comment by pboothe** [comment]
This also confused me.

**Comment by pboothe** [comment]
Either "Modeled maximum bandwidth" or "Expected Maximum Bandwidth" would have not confused me. When a maximum comes from a model, then it makes sense that the model updates over time. It also makes sense that an expected value would change over time.

**Comment by bassosimone** [comment]
> When a maximum comes from a model, then it makes sense that the model updates over time. It also makes sense that an expected value would change over time.

That's a very good argument! I believe we probably want different labelling depending on who's reading the data, but for engineers your suggestion seems very strong. (I failed to understand why it was so good until I've read the above quoted sentence, so thanks for explaining it.)

**Comment by elioqoshi** [comment]
I lack context and a proper understanding of the NDT7 environment to give proper UX advice at this point. Once I have a better look at it, I could chime in on this one I think :)

**Comment by bassosimone** [comment]
@elioqoshi indeed: I just meant to say that figuring out that is also part of that job!
